pkg/e2e/framework.go

```diff
@@ -30,6 +30,7 @@ import (
 	"testing"
 	"time"
 
+	cp "github.com/otiai10/copy"
```

Comment:
I wonder you could use https://github.com/moby/moby/blob/master/daemon/graphdriver/copy/copy.go#L124 as we already depend on moby and this would about an additional dependency

Reply:
I tried but kept on getting a very strange error:
```
# github.com/docker/docker/pkg/chrootarchive
../../../../pkg/mod/github.com/docker/docker@v24.0.6+incompatible/pkg/chrootarchive/archive_unix.go:31:8: undefined: goInChroot
../../../../pkg/mod/github.com/docker/docker@v24.0.6+incompatible/pkg/chrootarchive/archive_unix.go:53:8: undefined: goInChroot
../../../../pkg/mod/github.com/docker/docker@v24.0.6+incompatible/pkg/chrootarchive/diff_unix.go:45:8: undefined: goInChroot
FAIL    github.com/docker/compose/v2/pkg/e2e [build failed]
```

Reply:
I'd avoid the graphdriver packages as they are really intended for the daemon (and don't have macos implementations)

Perhaps (but typing from my phone) the copy package in fsutils could work? Is this just to copy a directory? https://github.com/tonistiigi/fsutil/tree/master/copy

Reply:
right, I just wondered, but that's ok
(I just miss the ability to define test-only dependencies in `go.mod`. I miss you `pom.xml` :)